androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/login/SplashScreen.kt

```diff
@@ -0,0 +1,66 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.login
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.Transparent
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.avishek.phong.kmmic.android.ui.screen.common.*
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.Black60
+import kotlinx.coroutines.delay
+
+private const val DELAY_LOGO = 500L
+private const val DURATION_LOGO = 1000
+
+@Composable
+fun SplashScreen() {
+    var shouldShowLogo by remember {
+        mutableStateOf(false)
+    }
+
+    LaunchedEffect(Unit) {
+        delay(DELAY_LOGO)
+        shouldShowLogo = true
+    }
+
+    SplashContent(shouldShowLogo = shouldShowLogo)
+}
+
+@Composable
+fun SplashContent(
+    modifier: Modifier = Modifier,
+    shouldShowLogo: Boolean,
+) {
```

Comment:
```suggestion
fun SplashContent(
    shouldShowLogo: Boolean,
	modifier: Modifier = Modifier,
) {
```

Reply:
Fixed in acd211abce503054711d0c24171bd83cc8117768